---a/net/minecraft/entity/ai/EntityAIAttackOnCollide.java
+++b/net/minecraft/entity/ai/EntityAIAttackOnCollide.java
@@ -6,8 +6,8 @@
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public class EntityAIAttackOnCollide extends EntityAIBase
-{
+public class EntityAIAttackOnCollide extends EntityAIBase {
+
     World worldObj;
     protected EntityCreature attacker;
     int attackTick;
@@ -21,14 +21,12 @@
     private double field_151496_k;
     private static final String __OBFID = "CL_00001595";
 
-    public EntityAIAttackOnCollide(EntityCreature p_i1635_1_, Class p_i1635_2_, double p_i1635_3_, boolean p_i1635_5_)
-    {
+    public EntityAIAttackOnCollide(EntityCreature p_i1635_1_, Class p_i1635_2_, double p_i1635_3_, boolean p_i1635_5_) {
         this(p_i1635_1_, p_i1635_3_, p_i1635_5_);
         this.classTarget = p_i1635_2_;
     }
 
-    public EntityAIAttackOnCollide(EntityCreature p_i1636_1_, double p_i1636_2_, boolean p_i1636_4_)
-    {
+    public EntityAIAttackOnCollide(EntityCreature p_i1636_1_, double p_i1636_2_, boolean p_i1636_4_) {
         this.attacker = p_i1636_1_;
         this.worldObj = p_i1636_1_.worldObj;
         this.speedTowardsTarget = p_i1636_2_;
@@ -36,84 +36,65 @@
         this.setMutexBits(3);
     }
 
-    public boolean shouldExecute()
-    {
+    public boolean shouldExecute() {
         EntityLivingBase entitylivingbase = this.attacker.getAttackTarget();
 
-        if (entitylivingbase == null)
-        {
+        if (entitylivingbase == null) {
             return false;
-        }
-        else if (!entitylivingbase.isEntityAlive())
-        {
+        } else if (!entitylivingbase.isEntityAlive()) {
             return false;
-        }
-        else if (this.classTarget != null && !this.classTarget.isAssignableFrom(entitylivingbase.getClass()))
-        {
+        } else if (this.classTarget != null && !this.classTarget.isAssignableFrom(entitylivingbase.getClass())) {
             return false;
-        }
-        else
-        {
+        } else {
             this.entityPathEntity = this.attacker.getNavigator().getPathToEntityLiving(entitylivingbase);
             return this.entityPathEntity != null;
         }
     }
 
-    public boolean continueExecuting()
-    {
+    public boolean continueExecuting() {
         EntityLivingBase entitylivingbase = this.attacker.getAttackTarget();
         return entitylivingbase == null ? false : (!entitylivingbase.isEntityAlive() ? false : (!this.longMemory ? !this.attacker.getNavigator().noPath() : this.attacker.func_180485_d(new BlockPos(entitylivingbase))));
     }
 
-    public void startExecuting()
-    {
+    public void startExecuting() {
         this.attacker.getNavigator().setPath(this.entityPathEntity, this.speedTowardsTarget);
         this.field_75445_i = 0;
     }
 
-    public void resetTask()
-    {
+    public void resetTask() {
         this.attacker.getNavigator().clearPathEntity();
     }
 
-    public void updateTask()
-    {
+    public void updateTask() {
         EntityLivingBase entitylivingbase = this.attacker.getAttackTarget();
         this.attacker.getLookHelper().setLookPositionWithEntity(entitylivingbase, 30.0F, 30.0F);
         double d0 = this.attacker.getDistanceSq(entitylivingbase.posX, entitylivingbase.getEntityBoundingBox().minY, entitylivingbase.posZ);
         double d1 = this.func_179512_a(entitylivingbase);
         --this.field_75445_i;
 
-        if ((this.longMemory || this.attacker.getEntitySenses().canSee(entitylivingbase)) && this.field_75445_i <= 0 && (this.field_151497_i == 0.0D && this.field_151495_j == 0.0D && this.field_151496_k == 0.0D || entitylivingbase.getDistanceSq(this.field_151497_i, this.field_151495_j, this.field_151496_k) >= 1.0D || this.attacker.getRNG().nextFloat() < 0.05F))
-        {
+        if ((this.longMemory || this.attacker.getEntitySenses().canSee(entitylivingbase)) && this.field_75445_i <= 0 && (this.field_151497_i == 0.0D && this.field_151495_j == 0.0D && this.field_151496_k == 0.0D || entitylivingbase.getDistanceSq(this.field_151497_i, this.field_151495_j, this.field_151496_k) >= 1.0D || this.attacker.getRNG().nextFloat() < 0.05F)) {
             this.field_151497_i = entitylivingbase.posX;
             this.field_151495_j = entitylivingbase.getEntityBoundingBox().minY;
             this.field_151496_k = entitylivingbase.posZ;
             this.field_75445_i = 4 + this.attacker.getRNG().nextInt(7);
 
-            if (d0 > 1024.0D)
-            {
+            if (d0 > 1024.0D) {
                 this.field_75445_i += 10;
-            }
-            else if (d0 > 256.0D)
-            {
+            } else if (d0 > 256.0D) {
                 this.field_75445_i += 5;
             }
 
-            if (!this.attacker.getNavigator().tryMoveToEntityLiving(entitylivingbase, this.speedTowardsTarget))
-            {
+            if (!this.attacker.getNavigator().tryMoveToEntityLiving(entitylivingbase, this.speedTowardsTarget)) {
                 this.field_75445_i += 15;
             }
         }
 
         this.attackTick = Math.max(this.attackTick - 1, 0);
 
-        if (d0 <= d1 && this.attackTick <= 0)
-        {
+        if (d0 <= d1 && this.attackTick <= 0) {
             this.attackTick = 20;
 
-            if (this.attacker.getHeldItem() != null)
-            {
+            if (this.attacker.getHeldItem() != null) {
                 this.attacker.swingItem();
             }
 
@@ -121,10 +121,8 @@
         }
     }
 
-    protected double func_179512_a(EntityLivingBase p_179512_1_)
-    {
-        return (double)(this.attacker.width * 2.0F * this.attacker.width * 2.0F + p_179512_1_.width);
+    protected double func_179512_a(EntityLivingBase p_179512_1_) {
+        return (double) (this.attacker.width * 2.0F * this.attacker.width * 2.0F + p_179512_1_.width);
     }
-}
+}
